phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/RepositoriesPanel.java

```diff
@@ -169,6 +169,7 @@ private void populateRepositoryRefresh() {
         this.refreshRepositoriesLabel.setWrapText(true);
 
         this.refreshRepositoriesButton = new Button("Refresh Repositories");
+        refreshRepositoriesButton.getStyleClass().add("buttonWithIcon");
```

Comment:
I would prefer, if this line starts with a `this.` because the other lines are written like this.

In addition I think we should define when we want to use the keyword `this` and when not, to do this uniformly.